dist/docs/guide/usage/add_feature_pack.adoc

```diff
@@ -0,0 +1,75 @@
+
+Sometimes feature packs may require additional artifacts whose lifecycle is separate from the feature pack itself. For example, Datasources feature pack lets users specify which version of database driver should be installed irrespective of the feature pack version. In such cases users have to create and subscribe to a new channel providing those artifacts.
+
+For example, a channel supplying Datasources feature pack could use https://repo1.maven.org/maven2[Maven Central] repository and a following manifest:
+```
+schemaVersion: 1.0.0
+streams:
+  - groupId: org.wildfly
+    artifactId: wildfly-datasources-galleon-pack
+    version: 4.0.1.Final
+  - groupId: com.mysql
+    artifactId: mysql-connector-j
+    version: 8.1.0
+```
+
+#### Installing feature pack
+
+The feature packs are identified by GroupId:ArtifactId part of their Maven coordinates. This is known as their Feature Pack Location (fpl).
+
+For example, assuming the server is subscribed to a datasources channel, following command will install the datasources feature pack:
+
+```
+$ prospero feature-pack add \
+  --fpl org.wildfly:wildfly-datasources-galleon-pack \
+  --dir wildfly
+```
+
+NOTE: This specific command does not in fact install any database drivers or datasources. See further examples bellow to see how to specify drivers and datasources to install or refer to https://github.com/wildfly-extras/wildfly-datasources-galleon-pack[wildfly-datasources documentation].
+
+#### Customizing installed feature packs
+
+The Galleon feature packs may provide different capabilities using https://docs.wildfly.org/galleon/#_layers[layers]. Users may select a set of layers to install using `--layers` parameter.
+
+NOTE: Using `--layers` parameter overwrites the default configuration defined by some feature packs.
+
+If the feature pack changes server configuration files, the user may also choose which server configuration should be affected using `--model` and `--config`
+parameters. If those parameters are not provided, the layer configuration will use default values. Most Wildfly feature packs provide only a single model (standalone) and the default configuration file of standalone.xml.
+
+NOTE: If the affected configuration file has been modified by the user, that file will not be modified. Instead, a new configuration file with suffix `.glnew` will be created with the changes.
+
+For example to install mysql support and configure the datasource in Wildfly `standalone-full.xml` configuration file, a following command can be used:
+
+```
+$ prospero feature-pack add \
+  --fpl org.wildfly:wildfly-datasources-galleon-pack \
+  --layers=mysql-datasource \
+  --config=standalone-full.xml \
+  --dir wildfly
+```
+
+#### Offline installation
+
+Similarly to other operations, `feature-pack add` can be executed in an offline mode. To do so, users should provide local repositories with all required artifacts (both for base server and the new feature) using `--repositories` parameter.
```

Comment:
I'm missing description of `--repositories` in the feature analysis. Can user also define a channel (with a manifest) for a new feature pack and then use `--repositories=<offline-feature-pack-channel-repo>,<offline-base-channel-repo>`?

Reply:
@honza-kasik yes the `--repositories` work in the same way as in `update` operation so the user would have to provide both base and feature pack repositories